Sources/Sentry/SentrySessionTracker.m

```diff
@@ -30,7 +31,9 @@ @interface SentrySessionTracker ()
 
 @end
 
-@implementation SentrySessionTracker
+@implementation SentrySessionTracker {
+    BOOL isStarted;
```

Comment:
`l`: Any specific reason not to define this as a regular property, like the ones above?

Reply:
No particular reason, refactoring it.